packages/documentation/src/stories/health/icons/icons.blocks.tsx

```diff
@@ -0,0 +1,104 @@
+// icons.blocks.tsx
+import React from 'react';
+import reportData from '../../../../node_modules/@swisspost/design-system-icons/public/report.json';
+import { IconReport } from './icons-report';
+
+const report = reportData as IconReport;
+
+export const StatusBlock: React.FC = () => (
+  <div className="status">
+    <strong>Version:</strong> {report.version}
+    <br />
+    <strong>Successful:</strong> {report.stats.success}
+    <br />
```

Comment:
What if we add some classes here (e.g. `text-success`, `text-warning`, `text-error`) to reflect, what kind of state it is?
But only when the amount in this specific state is bigger than zero.

I'll leave the decition to you if you implement this or not.

```suggestion
    {<div className={report.stats.successful.length ? 'text-success' : null}><strong>Successful:</strong> {report.stats.success}</div>}
```